scripts/chi-classnames/batch/classes.txt

```diff
@@ -0,0 +1,71 @@
+a-grid:chi-grid
```

Comment:
Would it be possible to use this file for both windows bat script and bash script?

Reply:
In case of Bash script the list is a native array. In this case it's actually a big string delimited with new line because Bat has a weird way of defining arrays with key-value pairs. I could use this file for both scripts but I think performance-wise would be better to use native array syntax that's currently used in Bash script. What do you think?